### ISSUE TITLE
Allow using nodemon with Visual Studio

### ISSUE THREAD

**Issue by rjamesnw** [microsoft/nodejstools#2192]
I noticed other posts on the issue, which are now quite old.  Is it still the stance that this will not be supported at all?  I'm trying to get a seemingly simple thing to work, which is to not have the Visual Studio (2017) debugger detach when nodemon restarts NodeJS.  I imagine it happens because the NodeJS process terminates and restarts.  Is there no way at all to have VS auto-reconnect? Is it really that big of a problem to support this?

**Comment by amcasey** [comment]
@rjamesnw Thanks for your feedback!  We think this might already be possible, but we're confirming before we post potentially incorrect details.

**Comment by amcasey** [comment]
Nope, turns out it doesn't work yet.  It sounds straightforward though.  @rakatyal, we probably just have to set [these parameters](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_restarting-debug-sessions-automatically-when-source-is-edited) [here](https://github.com/microsoft/nodejstools/blob/e13935445ecf9ce70e4a1b40762f692c1219575c/Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs#L262).

**Comment by rjamesnw** [comment]
Yeah, I already tested also before posting. ;) If this gets in it would significantly improve the development process by not having to build and run over and over. There is way too much overhead when having to stop and restart using VS.

**Comment by rakatyal** [comment]
@rjamesnw: Thanks for reporting the issue. I am in the process of figuring out what it would take on the VS side to support the "restart" option. Out of curiosity, can you tell me how are you currently integrating "nodemon" with Node Project System and launching it?

**Comment by rjamesnw** [comment]
There are two options:

1. Use NPM to save a local dependency on node non and use NodeJS to run the `nodemon.js` file. 

2. Install globally and run the `nodemon.cmd` command.

For both options I put the path to the command or JS file into the “path to NodeJS” project setting.

**Comment by rjamesnw** [comment]
Any updates? I just tried out of curiosity and it still doesn't work.  It would be great to have nodemon restart the NodeJS server automatically; would save a lot of overhead as Visual Studio spends a lot of time trying to build and launch for every simple change.
